rfcs/013-release_deployment_tracking/v1/README.md

```diff
@@ -0,0 +1,181 @@
+# Version 1 
+
+## Implementation from 24/01/2019
+
+This solution describes a set of steps to track build, release and deployment but is deliberately agnostic of deployment mechanism. 
+
+We make use of SSM to track which release should deploy into which environment. Out terraform stacks read from these SSM parameters to get which images to apply.
+
+Terms defined in this document apply only here.
+
+### Build artifacts
+
+> **Build artifact:** A container image encapsulating application code that can be configured by environment variable. The artifact _does not_ contain configuration.
```

Comment:
Suggestion: move the phrase about environment variable until after the second sentence. It doesn't make sense to talk about how it's configured until we know the image itself doesn't contain configuration.

```suggestion
> **Build artifact:** A Docker image containing our application code. An artifact does not contain configuration; instead, it can be configured by passing environment variables.
```